### PR TITLE
Cleans up response section, adds examples

### LINKED ISSUE #1627: Implement docs > linodes > endpoint > GET section response

### PR DESCRIPTION
**Pull request by mparke**
closes #1627 

followup for inlined things: https://github.com/linode/manager/issues/1651

Example:
screen shot 2017-04-27 at 4 04 32 pm

Other updates that effect manager:

- Tables are now `table-layout: fixed`
- `disableHeader` now adds a class to `thead` instead of not rendering `th`. Now `th` are rendered in the head but with `height: 0` so that they can still be given a width for a column

### CODE REVIEW

File: components/scss/components/tables/Table.scss [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
-    th:first-of-type,
-    td:first-of-type {
-        padding-left: 15px;
-    }
-
-    td:last-of-type {
-        padding-right: 15px;
+    th, td {
+        padding: 5px 10px;
```
Comment (eatonphil) [inline components/scss/components/tables/Table.scss]: lr padding 15px plz?

**Comment by mparke** [comment]
Follow up related to table actions: https://github.com/linode/manager/issues/1665

File: components/scss/components/tables/Table.scss (hunk 1 above)
Comment (mparke) [inline components/scss/components/tables/Table.scss]: Updated padding

**Comment by eatonphil** [comment]
The table text is too small to read. Also, we've got some bleeding on larger fields:
screen shot 2017-05-01 at 2 20 46 pm

**Comment by mparke** [comment]
will receive another round of styles after, if any columns are overflowing currently

**Comment by mparke** [comment]
tables should be o.k. manager side with the `table-layout: fixed` change now, but let me know if anyone sees any breaks

File: src/linodes/layouts/IndexPage.js [hunk 2]
```diff
@@ -155,17 +155,18 @@ export class IndexPage extends Component {
...
                       hrefFn: (linode) => `/linodes/${linode.label}`,
                     },
-                    { cellComponent: IPAddressCell },
-                    { cellComponent: RegionCell },
+                    { cellComponent: IPAddressCell, headerClassName: 'IPAddressColumn' },
+                    { cellComponent: RegionCell, headerClassName: 'RegionColumn' },
                     {
                       cellComponent: BackupsCell,
                       className: 'hidden-md-down',
+                      headerClassName: 'BackupsColumn',
```
Comment (eatonphil) [inline src/linodes/layouts/IndexPage.js]: Make this `'BackupsColumn hidden-md-down'` and you've got it

**Comment by eatonphil** [comment]
btw there is a hidden-md-down class in the support ticket list. Could you make sure it gets all the header love it needs there too?

**Comment by mparke** [comment]
Tickets now have `th`, and the collapse of the id works correctly. Suffers from the same right border loss as other places though.

**Comment by mparke** [comment]
Follow up for table border:

https://github.com/linode/manager/issues/1682

File: components/tables/Table.js [hunk 3]
```diff
@@ -17,16 +17,12 @@ export default function Table(props) {
...
-      <thead>
-        <TableHeaderRow columns={columns} />
-      </thead>
-    );
-  } else {
-    tableHeader = (<thead></thead>);
-  }
+  const theadClassName = disableHeader ? 'TableHead--disabled' : '';
+  const tableHeader = (
+    <thead className={`${theadClassName}`}>
```
Comment (eatonphil) [inline components/tables/Table.js]: this can just be `className={theadClassName}`

File: docs/package.json [hunk 4]
```diff
@@ -49,13 +49,14 @@
...
     "bootstrap": "^4.0.0-alpha.6",
+    "clipboard": "^1.6.1",
     "fontawesome": "^4.5.0",
     "history": "^2.1.2",
     "linode-components": "file:../components",
     "lodash": "^4.16.4",
     "moment": "^2.17.1",
     "moment-timezone": "^0.5.11",
-    "react": "^15.1.0",
+    "react": "https://registry.npmjs.org/react/-/react-15.5.4.tgz",
```
Comment (eatonphil) [inline docs/package.json]: Why the change?

File: docs/scss/components/Method.scss [hunk 5]
```diff
@@ -1,7 +1,80 @@
...
+                margin-right: 5px;
+                padding: 4px 10px;
+
+            }
+
+            &-container {
+                margin: 0;
+                padding: 0;
+            }
+        }
```
Comment (eatonphil) [inline docs/scss/components/Method.scss]: re ReactTabs__TabList & co: If we have our own wrapper classNames around theirs, could you use those instead?

File: docs/src/components/Tables/Cells/DescriptionCell.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+
+export default function DescriptionCell(props) {
+  const { column, record } = props;
+
+  return (
+    <td className={`Table-cell DescriptionCell`}>
+      <div>
+        {record.description}
+      </div>
```
Comment (eatonphil) [inline docs/src/components/Tables/Cells/DescriptionCell.js]: Is the extra div needed?

File: docs/src/components/Tables/Cells/FieldCell.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  const { column, record } = props;
+
+  return (
+    <td className={`Table-cell FieldCell`}>
+      <div>
+        {record.name}
+      </div>
+      {/*<div className="FieldCell-type">*/}
+        {/*<small><i>{record.type}</i></small>*/}
+      {/*</div>*/}
```
Comment (eatonphil) [inline docs/src/components/Tables/Cells/FieldCell.js]: Delete the comment stuff?

File: docs/src/components/Tables/Cells/FieldCell.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+
+// { label: 'Type', dataKey: 'type', headerClassName: 'TypeColumn' },
```
Comment (eatonphil) [inline docs/src/components/Tables/Cells/FieldCell.js]: Replace comments with propTypes?

File: docs/src/components/Tables/Cells/DescriptionCell.js (hunk 6 above)
Comment (mparke) [inline docs/src/components/Tables/Cells/DescriptionCell.js]: fixed

File: docs/src/components/Tables/Cells/FieldCell.js (hunk 7 above)
Comment (mparke) [inline docs/src/components/Tables/Cells/FieldCell.js]: removed

File: docs/src/components/Tables/Cells/FieldCell.js (hunk 8 above)
Comment (mparke) [inline docs/src/components/Tables/Cells/FieldCell.js]: prop types added

File: docs/scss/components/Method.scss (hunk 5 above)
Comment (mparke) [inline docs/scss/components/Method.scss]: we do, and it's being used here
